src/app/components/SearchResults/SearchResults.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import ResultsMetadata from './ResultsMetadata';
 import ResultsList from './ResultsList';
 import SearchHeader from './SearchHeader';
+import SearchFooter from './SearchFooter';
```

Comment:
Wondering if there couldn't be some shared components between SearchHeader and SearchFooter treatments. Are there specific things that are different now for both or would this purely be for future implementation of unique elements for this content bar? If you could share some things, that would help with minimizing repetition of code.

Reply:
I began using the SearchHeader with some flag, but quickly found out that it was a very artificial  difference that could lead to confusion in the future. There are some common components that could be separated